docs/ep-01-index-representation.md

```diff
@@ -0,0 +1,193 @@
+# EP-01: Internal State-Action-Space Representation
+
+## Abstract
+
+This LEP outlines the use of an internal index representation for states and actions to
+address two challenges. First, it enables the implementation of discrete actions with a
+time-dependent set of valid actions, thereby reducing the state–action space in certain
+periods. Second, it supports continuous stochastic states and allows continuous states
+and actions to influence stochastic transitions.
+
+## Backwards Compatibility
+
+All changes are fully backward compatible.
+
+## Motivation
+
+To Be Written.
+
```

Comment:
```suggestion
```
Irrelevant / obvious. Don't waste time on fitting a scheme that does not fit yet.